src/main/java/net/sf/jabref/logic/util/version/Version.java

```diff
@@ -0,0 +1,98 @@
+/*  Copyright (C) 2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.logic.util.version;
+
+import java.util.regex.Pattern;
+
+import net.sf.jabref.logic.util.BuildInfo;
+
+/**
+ * Represents the Application Version with the major and minor number, the full Version String and if it's a developer version
+ */
+public class Version {
+
+    private String fullVersion;
+    private int major;
+    private int minor;
+    private boolean devVersion;
+
+    /**
+     * @param version cannot be empty
+     */
+    public Version(String version) {
+        if (version.equals("")) {
+            this.fullVersion = BuildInfo.UNKOWN_VERSION;
+            this.major = this.minor = 0;
+            this.devVersion = false;
+        } else {
+            this.fullVersion = version;
+            String[] versionParts = version.split(Pattern.quote("."));
+            major = Integer.parseInt(versionParts[0]);
+            minor = Integer.parseInt(versionParts[1].split("dev")[0]);
```

Comment:
I added some test cases for Version.
The version now only has to own the major part, the minor one is optional.